examples/pytorch/cluster_gcn/run_ppi.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+
+python cluster_gcn.py --gpu 0 --dataset ppi --lr 1e-2 --weight-decay 0.0 --psize 50 --batch-size 2 --n-epochs 300 --n-hidden 1024 --n-layers 1 --log-every 100 --use-pp --self-loop --note self-loop-ppi-non-sym-ly3-pp-cluster-2-2-wd-5e-4 --dropout 0.0 --model-type graphsage --use-val --normalize
```

Comment:
What are the experimental results?

https://github.com/google-research/google-research/tree/master/cluster_gcn#run-experiments

Reply:
My bad that I forgot to mention. The original repo replicates the results of the paper, which is around 0.994. However, it uses some tricks about enhancing adj diagonal, to retain locality when stacking deep conv net. It might not be worth it to implement it for Reddit since it would not be sota anyway, but make sense to see how it works on PPI. I was not focusing on the performance but rather the speed before, but now I will add a milestone on this. Though I would not have time to work on this before this weekend.

Reply:
New PPI results appended